NBitcoin/Secp256k1/ECPubKey.cs

```diff
@@ -304,6 +305,32 @@ private bool secp256k1_eckey_pubkey_tweak_add(ECMultContext ctx, ref GE key, in
 			return true;
 		}
 
+		/// <summary>
+		/// The original function name is `secp256k1_ec_pubkey_combine`
+		/// </summary>
+		public static bool TryCombine(Context ctx, ECPubKey[] pubkeys, out ECPubKey? combinedPubKey)
+		{
+			if (pubkeys == null)
+				throw new ArgumentNullException(nameof(pubkeys));
+
+			combinedPubKey = null;
+			var n = pubkeys.Count();
```

Comment:
You must use `.Length` because `.Count()` is actually walking over the whole collection to count stuff.

Reply:
Ok. I actually thought `Count()` will become alias for `Length` in case of  arrays.

Reply:
actually Joe is right here: https://stackoverflow.com/a/981292/544947